mautrix_googlechat/db/message.py

```diff
@@ -49,7 +49,7 @@ def _from_row(cls, row: Record | None) -> Message | None:
 
     columns = (
         "mxid, mx_room, gcid, gc_chat, gc_receiver, gc_parent_id, "
-        "index, timestamp, msgtype, gc_sender"
+        "`index`, timestamp, msgtype, gc_sender"
```

Comment:
```suggestion
        '"index", timestamp, msgtype, gc_sender'
```

It's better to use `"` as the delimiter within the SQL, and switch to `'` for the Python string delimiter.

According to https://stackoverflow.com/questions/34884362/escape-table-name-in-sqlite#34884445:

> `keyword` A keyword enclosed in grave accents (ASCII code 96) is an identifier. This is not standard SQL. This quoting mechanism is used by MySQL and is included in SQLite for compatibility.

(This applies to the entire PR)

Reply:
Done throughout.